manifests/v1beta1/components/controller/katib-config.yaml

```diff
@@ -57,6 +57,19 @@ data:
       },
       "darts": {
         "image": "docker.io/kubeflowkatib/suggestion-darts:latest"
+      },
+      "pbt": {
+        "image": "docker.io/kubeflowkatib/suggestion-pbt:latest",
+        "persistentVolumeClaimSpec": {
+          "accessModes": [
+            "ReadWriteMany"
```

Comment:
```suggestion
            "ReadWriteOnce"
```

When I reproduced E2E errors on my local machine, suggestion-pbt is Pending since It seems that the KinD storage provisioner only supports "ReadWriteOnce".

ref: https://github.com/kubernetes-sigs/kind/issues/2371

```shell
$ kubectl get pvc --n kubeflow
NAMESPACE   NAME             STATUS    VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
kubeflow    katib-mysql      Bound     pvc-ce5b364d-12b4-4681-9def-4b89d8a17004   2Gi        RWO            standard       7m19s
kubeflow    simple-pbt-pbt   Pending                                                                        standard       5m10s
```

I changed the mode of the PVC and PV was generated.

```shell
$ kubectl get pvc -n kubeflow
NAMESPACE   NAME             STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
kubeflow    katib-mysql      Bound    pvc-ce5b364d-12b4-4681-9def-4b89d8a17004   2Gi        RWO            standard       14m
kubeflow    simple-pbt-pbt   Bound    pvc-c3b71cdf-7b30-4f93-8631-67c6260752d6   5Gi        RWO            standard       7s
```

```shell
$ kubectl get cm -n kubeflow katib-config -ojsonpath='{.data}' | jq -r '.suggestion'
{
  "pbt": {
    "image": "docker.io/kubeflowkatib/suggestion-pbt:e2e-test",
    "persistentVolumeClaimSpec": {
      "accessModes": [
        "ReadWriteOnce"
      ],
      "resources": {
        "requests": {
          "storage": "5Gi"
        }
      }
    }
  }
}
```

Do we need "ReadWriteMany" PVC in suggestion-pbt?

Reply:
I don't think we need to change the `run-e2e-experiment.go`, what you pointed out here is the main issue! The suggestion mount should have RWX access as it is a shared mount across the running trials + suggestion service. For CI testing, RWO should be fine though; is there any way to flag this mount mode for testing vs general use?

Reply:
@a9p What is the differences running PBT during CI and manually ?
Will your Experiment fail with RWO ?

Reply:
Actually, I believe the experiment should always fail with RWO (CI or otherwise). Suggestion service should always have the claim to the volume so the spawned Trials will get stuck at pending while trying to get the subdirectory mount.

Reply:
I don't think Kind supports this out of the box as @tenzen-y mentioned above.
So we might consider add more unit/integration tests for PBT Suggestion and Trial service until Kind supports RWX.

Reply:
Thank you for investigating this @tenzen-y!
Do we have any limitations in our CI with Minikube ?
Do you know, is it possible to use Kind with different volume provisioner ( it seems they are using [Local Path Provisioner](https://github.com/kubernetes-sigs/kind/issues/2371#issuecomment-880633851) from Rancher) ?

Reply:
> Do we have any limitations in our CI with Minikube ?

Maybe, we do not have any limitations, but minikube cluster is a bit more heavy cluster rather than the KinD cluster.
We may face computing resource limitations of GH Actions.

> Do you know, is it possible to use Kind with different volume provisioner ( it seems they are using https://github.com/kubernetes-sigs/kind/issues/2371#issuecomment-880633851 from Rancher) ?

Yes, we can use other provisioners in the KinD cluster (ex. [topolvm](https://github.com/topolvm/topolvm)). 

However, it might be better to use RWO PVC in that use case since our CI system launch only one node.

What do you think? @andreyvelich @johnugeorge

Reply:
looks good

Reply:
@tenzen-y did you see if Minikube works?  Isn't that the only way forward?

Reply:
@johnugeorge 
I have not tried Minikube yet.
I will see it by today (UTC).

Reply:
FYI: https://github.com/kubeflow/katib/pull/1899